influxdb/etc/influxdb.conf

```diff
@@ -1,7 +0,0 @@
-[meta]
-  dir = "/var/lib/influxdb/meta"
```

Comment:
I would leave this out, these are defaults which work great within docker. Just have to map volumes correctly though